tensorflow/core/kernels/matmul_op.cc

```diff
@@ -276,13 +276,20 @@ struct MatMulFunctor<CPUDevice, T> {
                               .Label("cublas"),                    \
                           MatMulOp<GPUDevice, T, true /* cublas */>)
 
+#if defined (INTEL_MKL)
```

Comment:
This change is part of the matmul pull request and doesn't need to be reviewed here.